pkg/daemon/update.go

```diff
@@ -119,8 +119,8 @@ func (dn *Daemon) finalizeAndReboot(newConfig *mcfgv1.MachineConfig) (retErr err
 }
 
 func (dn *Daemon) drain() error {
-	// Skip draining of the node when we're not cluster driven
-	if dn.kubeClient == nil {
+	// Skip draining of the node when we're not cluster driven or SNO mode(skip always for now)
+	if dn.kubeClient == nil || true {
```

Comment:
right now it will skip drain for all cases. Did you mean to add some logic here specific to SNO?

Reply:
My understanding is that this PR is being used to build temporary release images and to highlight the sections of code that need changes, but that it isn't meant to be merged as-is.

I expect this would also look at the `highAvailabilityMode` flag mentioned in the comment from @cgwalters above and described in https://github.com/openshift/enhancements/pull/555